core-blocks/latest-comments/edit.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import {
+	Disabled,
+	PanelBody,
+	RangeControl,
+	ToggleControl,
+	ServerSideRender,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import {
+	InspectorControls,
+	BlockAlignmentToolbar,
+	BlockControls,
+} from '@wordpress/editor';
+
+/**
+ * Internal dependencies.
+ */
+import './editor.scss';
+
+const MIN_COMMENTS = 1;
```

Comment:
IMO we should JSDoc any constant, even just to establish the convention in encouraging the practice for less-obvious constants.

Reply:
I'm cool with that 👍